deployment_v2.md

```diff
@@ -0,0 +1,122 @@
+# Montage Deployment
+
+These are instructions for deploying Montage on Toolforge.
+
+## Deploying on Toolforge from scratch
+These instructions is only first time when setuping project on Toolforge
+
+##### 1. Get thee OAuth credentials.
+[Register your app](https://meta.wikimedia.org/wiki/Special:OAuthConsumerRegistration/propose) and save your consumer token and secret token for later.
+
+##### 2. SSH to Toolforge and then inside tool
+```bash
+ssh <shell-username>@login.toolforge.org
+become montage-beta
+```
+Here, we are using `montage-beta` instance but it can be `montage` or `montage-dev` as well.
+
+##### 3. Clone the repo as src directory
+```bash
+mkdir -p $HOME/www/python
+cd $HOME/www/python
+git clone https://github.com/hatnote/montage.git src
+```
+
+##### 4. Make the frontend build
+```bash
+toolforge webservice node18 shell -m 2G
+cd $HOME/www/python/src/frontend
+npm install
+npm run toolforge:build
```

Comment:
Just thinking about how often things can change/break on toolforge, I'm wondering if it makes sense to document a way to build a production version locally and upload it using simple tools (without needing a whole node env)?

Reply:
I played with fabric.I think we can easily zip the bundle locally and upload it to the tool with a webservice restart as we don't need to go inside pos.